tests/test_sensor.py

```diff
@@ -384,6 +385,86 @@ def test_silicon_fft():
     print('check |r1-r3| = %f >? %f'%(np.abs(r1-r3), 2.*sigma_r))
     assert r1-r3 > 2.*sigma_r
 
+@timer
+def test_silicon_area():
+    """Test the Silicon class calculate_pixel_areas() function.
```

Comment:
I ran a test where I ran a small (sigma = 0.02) spot with 80,000 electrons, which is what I used in the Poisson_CCD code to build the pixel distortion file, and then compared the areas between GalSim and Poisson_CCD.  They agree very well - to better than 0.01%.  I could add this to the unit test, but then I would have to add the area file to share/sensor, so I have this to compare to.  Let me know if you want me to do this.
 Let me know if you want me to do this.

Reply:
That sounds good.  Don't add that file to share/sensor though.  Rather, put it somewhere in the tests directory.  E.g. a new subdirectory tests/sensor_validation perhaps.  You'll see there are lots of other similar directories there that serve a similar purpose, validating against various kinds of external sources of truth.